net/ptunnel-ng/Makefile

```
@@ -32,6 +32,9 @@ CONFIGURE_ARGS += \
 	--disable-pcap \
 	--disable-selinux
 
+CONFIGURE_VARS += \
+	ac_cv_header_bsd_stdlib_h=no
```

```suggestion
	ac_cv_header_bsd_stdlib_h=no \
	ac_cv_search_arc4random=no
```
It will still look for the library.  So disable that as well.  It is not without security/performance implications, though.  The package uses libbsd for the `arc4random` function.  If not found, it reads from `/dev/random`, or else uses `rand()`.

---

Fascinating... I looked at this package previously. It's coming back to me. Thank you for the review.

I just looked again after those changes and HAVE_RANDOM does not appear to be defined. Will investigate.

---

Oh I see, it is defined. it's a part of the make flags instead of the config file.

```
-DHAVE_RANDOM=1
```

---

The `pt_random()` function gets called when

1. a new connection is accepted by the proxy (unique 16 bit id for the proxy descriptor chain)
2. a challenge is generated (only used in conjunction with `--passwd`)

I have concerns regarding the entropy when using `/dev/urandom` for the challenge generation.